### DISCUSSION TITLE
Public project not appearing in the community projects list in qfieldcloud

### DISCUSSION THREAD

**Discussion by robinjonesuk** [opengisch/QField#5885] [Q&A]
Hi,
I'm sure that previously my project https://app.qfield.cloud/a/robinjones/River_Frome_Balsam_Tracker/ appeared in the community projects list, but is no longer showing. "Is Public" is definitely ticked in the settings, but searching or browsing through the community projects list does not locate it.

Does anyone know what I need to do to make it appear?

Thanks,
Robin

**Comment by SeqLaz** [comment]
Hi @robinjonesuk ,

I conducted a test, and the project results appeared correctly. You just need to allow some time for the search. It's also recommended to type the project name as it appears in QFieldCloud, including the underscores.

![image](https://github.com/user-attachments/assets/975b0e50-5563-4fdb-a421-135be003435e)

**Comment by robinjonesuk** [comment]
Hi,
thanks for checking this out. For some strange reason it behaves differently for me and will not locate this project. This is after waiting for the full refresh of the community projects list and using a very recent version of qfield (Android Pie) and searching for:
River_Frome_Balsam_Tracker

The search results go empty as soon as I type the first underscore. 

Thanks anyway.

Robin

**Reply by SeqLaz** [reply]
Hi @robinjonesuk 

Yes, after you type the name need to give more time for the search of the project,

How long did you wait?

**Comment by robinjonesuk** [comment]
It must have been at least ten minutes. I did notice that there was still internet activity. Was that still too little to wait?

**Comment by robinjonesuk** [comment]
I hadn't realised that public projects do not appear in the cloud projects list if they are owned by you or for which you are a collaborator - i.e. already in 'My Projects' tab. I was trying to explain to someone how to find the project and when I couldn't locate it, assumed they wouldn't be able to either. 
Thankyou,
Robin
